op-node/chaincfg/chains.go

```diff
@@ -0,0 +1,49 @@
+package chaincfg
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum-optimism/optimism/op-node/eth"
+	"github.com/ethereum-optimism/optimism/op-node/rollup"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+var Beta1 = rollup.Config{
+	Genesis: rollup.Genesis{
+		L1: eth.BlockID{
+			Hash:   common.HexToHash("0x87ba22412f6d081a28ca0d8aafcf630d13a0d2fc16a7345eb3a0d5cd329f935e"),
+			Number: 7996739,
+		},
+		L2: eth.BlockID{
+			Hash:   common.HexToHash("0x76aac93f04b04b051c0232dc96cdfb2ebd150ce726aa9f776a4713c2ac524dc8"),
+			Number: 0,
+		},
+		L2Time: 1669088016,
+		SystemConfig: eth.SystemConfig{
+			BatcherAddr: common.HexToAddress("0xc02551cde892e9716363b8e99d655298909e1a84"),
+			Overhead:    eth.Bytes32(common.HexToHash("0x0000000000000000000000000000000000000000000000000000000000000834")),
```

Comment:
nit: overhead and scalar should be as human readable as possible here

Reply:
I think leaving this in hex is a bit clearer - otherwise, we'll have to either update the type to something numeric or convert the number into a hex-padded `bytes32`. Right now it matches the rollup config and can be easily verified.